src/box/lua/console.lua

```diff
@@ -278,6 +278,16 @@ local function set_language(storage, value)
         local msg = 'Invalid language "%s", supported languages: lua and sql.'
         return error(msg:format(value))
     end
+    if value == 'sql' then
+        local is_configured = false
+        for k, _ in pairs(box) do
+            is_configured = is_configured or k == 'execute'
+        end
```

Comment:
Thank you for the review! I can't use `box.execute ~= nil` because accessing the `nil` field of `box` throws an error:
```
tarantool> box.something
---
- error: "builtin/box/load_cfg.lua:864: Please call box.cfg{} first\nstack traceback:\n\tbuiltin/box/load_cfg.lua:864:
    in function '__index'\n\t[string \"return box.something\"]:1: in main chunk\n\t[C]:
    in function 'pcall'\n\tbuiltin/box/console.lua:422: in function 'eval'\n\tbuiltin/box/console.lua:733:
    in function 'repl'\n\tbuiltin/box/console.lua:784: in function <builtin/box/console.lua:772>"
...

tarantool> box.something ~= nil
---
- error: "builtin/box/load_cfg.lua:864: Please call box.cfg{} first\nstack traceback:\n\tbuiltin/box/load_cfg.lua:864:
    in function '__index'\n\t[string \"return box.something ~= nil\"]:1: in main chunk\n\t[C]:
    in function 'pcall'\n\tbuiltin/box/console.lua:422: in function 'eval'\n\tbuiltin/box/console.lua:733:
    in function 'repl'\n\tbuiltin/box/console.lua:784: in function <builtin/box/console.lua:772>"
...
```

Reply:
`rawget()` should do the trick.

Reply:
Thank you! Fixed.

Reply:
*(I'm fine with the fix, but I left the discussion open, because it was opened by Vlad.)*